hail/Makefile

```diff
@@ -22,14 +22,18 @@ $(eval $(call ENV_VAR,BRANCH))
 $(eval $(call ENV_VAR,URL))
 $(eval $(call ENV_VAR,SPARK_VERSION))
 $(eval $(call ENV_VAR,HAIL_PIP_VERSION))
+$(eval $(call ENV_VAR,HAIL_DEBUG_MODE))
 
 HAIL_PYTHON3 ?= python3
 PIP ?= $(HAIL_PYTHON3) -m pip
 
 # not perfect, not robust to simdpp changes, but probably fine
+BUILD_DEBUG_PREFIX := build/classes/scala/debug
 JAR_SOURCES := $(shell git ls-files src/main)
 JAR_SOURCES += build.gradle
 JAR_TEST_SOURCES := $(shell git ls-files src/test)
+JAR_DEBUG_SOURCES := src/debug/scala/is/hail/annotations/Memory.java
```

Comment:
For the record, I did this manually because I don't actually like the `git ls-files` shell out that I originally did to get this quick and correct. I don't like how `git ls-files` means that you can't build this from a source tarball.